.github/workflows/canary-ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Canary CI
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  parse-version:
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/canary-ci run') }}
+    runs-on: ubuntu-latest
+    outputs:
+      react_version: ${{ steps.get-version.outputs.react_version }}
```

Comment:
Do we need this?

Reply:
my understanding is that this is how github is able to know to store this value so it's available in the other jobs to be referenced as `react_version: ${{ needs.parse-version.outputs.react_version }}`

Reply:
https://github.com/pmndrs/jotai/blob/93f2e16a4b4976eaf060273ea293d07c9d6e6d61/.github/workflows/ecosystem-ci.yml is my work. I didn't use `outputs:` and it seems working. Can you try removing it?